src/stdlib/WebSharper.Testing/Pervasives.fs

```diff
@@ -124,6 +124,9 @@ type TestCategoryBuilder(name: string) =
     [<Inline>]
     member this.Zero() = ()
 
+    [<Inline>]
+    member this.Combine(_, x) = x

```

Comment:
This was an attempt at supporting `if` in the CE, but IIRC it didn't work. If the solution still builds without this then this should be removed.